src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -89,6 +89,9 @@ private boolean setUpIntakeWheelSparks() {
 
     errors += SparkMaxUtils.check(deployMotor.setIdleMode(IdleMode.kCoast));
     errors += SparkMaxUtils.check(deployMotor.setIdleMode(IdleMode.kCoast));
+    //invert stuff
+    deployMotor.setInverted(true);
+    intakeRight.setInverted(true);
```

Comment:
This one is already following left inverted so it's not needed. We can maybe group inversion code next to follow code to make this a little clearer